flight/Modules/OnScreenDisplay/onscreendisplay.c

```diff
@@ -715,13 +717,17 @@ void draw_flight_mode(int x, int y, int xs, int ys, int va, int ha, int flags, i
 			write_string("TAB POI", x, y, xs, ys, va, ha, flags, font);
 			break;
 		}
+		}
```

Comment:
The indentation of the inner block is borked.

Reply:
How so?

Reply:
(We tend not to indent for {} in case because otherwise you run out of tabstops reaaaal quick)

Reply:
In the diff for this PR is a counterexample :stuck_out_tongue_closed_eyes: https://github.com/d-ronin/dRonin/pull/2152/files/ef9fe7779342502133b63e00e8dd226f7ca3d013..35a076e2be96b621baa58b181ad05e69929c6f11#diff-4f4804c22880c52150730a11f3c4a5baR1177

Reply:
I'm saying the {} after a label.

I don't really favor the {} after a label, preferring (void) 0; to avoid bracket weirdness.  Would you prefer that?

Reply:
I don't really care too much tbh. It just looks weird with two scopes closing at the same indentation.... Maybe we should just write everything in Python? :grimacing:

Reply:
I went with the (void) 0; trick to get rid of one.